chia/rpc/wallet_rpc_api.py

```diff
@@ -1339,22 +1340,33 @@ async def nft_get_nfts(self, request) -> Dict:
 
     async def nft_transfer_nft(self, request):
         assert self.service.wallet_state_manager is not None
-        wallet_id = int(request["wallet_id"])
+        wallet_id = uint32(request["wallet_id"])
```

Comment:
We can track this separately, but we still need to add support for specifying a fee to push the transaction through.